src/test/java/com/selfxdsd/selfweb/api/output/JsonProjectTestCase.java

```diff
@@ -0,0 +1,578 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
```

Comment:
@Lumi3011 Can you copy out licence header here? Just look in any other file and copy it here instead of this placeholder.